release-doc-in-process/all_vf.TTL

```diff
@@ -22,13 +23,18 @@ vf:Action  a           owl:Class ;
         rdfs:subClassOf   rdf:Property ;
         rdfs:comment   "An action verb defining the kind of event, commitment, or intent." .
 
+#vf:AgentRelationshipRole  a  owl:Class ;
```

Comment:
Because @elf-pavlik prefers to just have the relationship be a rdf:Property, instead of creating our own subclass of the rdf:Property, and I have agreed to that.  The name would remain in the UML for non-LOD implementations.